### ISSUE TITLE
AutoSearchInput component does not have a valid label

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-frontend#6682]
The `AutoSearchInput` does not have a valid label according to the WAVE a11y tool because there is no link between the `input` and the `label`. That's because we do not use the `id` attribute but a `name` attribute to link both HTML tags.

**Comment by ioanarusiczki** [comment]
@willdurand Please add some STR for me the qa 
Thanks!

**Comment by willdurand** [comment]
I don't think this needs QA, so I marked the issue as "qa: not needed", thanks!
